src/public/images/1/0x83F20F44975D03b1b09e64809B757c47f942BEeA/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "removed": false,
+  "address": "0x83f20f44975d03b1b09e64809b757c47f942beea",
+  "symbol": "sDAI",
+  "name": "Savings DAI",
+  "logoURI": "https://raw.githubusercontent.com/cowprotocol/token-lists/main/src/public/images/1/0x83F20F44975D03b1b09e64809B757c47f942BEeA/logo.png",
+  "reason": "New Gnosis token\r\n\r\nAs per @avsavsavs request",
```

Comment:
This is the Maker SPARK token, right? nothing to do with Gnosis (other than, there's an sDAI in Gnois Chain which is different)